CHANGES.rst

```diff
@@ -17,6 +17,9 @@ Bug fixes:
 - Remove outgoing 'isReferencing' RelationValues from catalog on deleting content item.
   [ksuess] (#93)
 
+- Report breaches once per item in delete_confirmation_info
```

Comment:
You added this under an already released version.
Please put this in a new file instead, named something like `news/95.bugfix`.

Reply:
@jaroel needs to adapt to our new ways. You have been out of the loop for way too long my friend. :)

Reply:
I should know better. Thanks!